app/views/availabilities/_availabilities_table.html.erb

```diff
@@ -16,9 +16,9 @@
       <!-- Full available -->
     <% (availabilities + partial_availabilities).each do |a| %>
       <tr <%== availability_status_class(a, @event) %>>
-        <td><%= Person::TITLE_ORDER[a.person.title] %></td>
+        <td><%= Person::TITLE_ORDER[a.person.shortrank] %></td>
```

Comment:
I think this will introduce a bug in click to sort. This line is looking into that hash. The keys are title, not short rank, so it will never find the corresponding value.

Reply:
@kgf got it... sorry, a little new to Git here. Do I need to make another PR with the changes to revert that issue, or can I do that through Github?